seqr/views/utils/test_utils.py

```diff
@@ -51,6 +53,9 @@ def setUp(self):
         self.mock_analyst_group.resolve_expression.return_value = 'analysts'
         self.addCleanup(patcher.stop)
 
+        self._log_stream = StringIO()
+        logging.getLogger().handlers[0].stream = self._log_stream
```

Comment:
Good to know we can do that without mocking.

Reply:
yeah took a while to figure this one out!